Fixing screen reader heuristics issues with examples in "Headers attribute specified on a cell refers to cells in the same table element" (#1505)
act-rules/act-rules.github.io: _rules/table-headers-attribute-refer-to-data-cells-a25f45.md

## Expectation 1

> Each target's `headers` [attribute value][] is a [set of space separated tokens][]. Each token is the value of the `id` attribute of an element, that is a [`cell`][] of the same [`table`][].

Reviewer: The target is already the `headers` attribute.
```suggestion
Each target's [attribute value][] is a [set of space separated tokens][]. Each token is the value of the `id` attribute of an element, that is a [`cell`][] of the same [`table`][].
```

**Note:** `headers` attribute referencing elements that are non-existent or not in the table are ignored when [assigning header cells (step 3, first case, point 2)](https://html.spec.whatwg.org/multipage/tables.html#algorithm-for-assigning-header-cells).

## Expectation 2

> Each target's `headers` [attribute value][] is a [set of space separated tokens][] that is not the value of `id` attribute of the element on which the test target is specified.

Reviewer: We need to keep the "none of which" formulation. `<td id="foo" headers="foo bar">`, the set {`foo`, `bar`} is not the value `foo`, so that would pass the expectation.
```suggestion
Each target's [attribute value][] is a [set of space separated tokens][], and none of these tokens is the value of `id` attribute of the element on which the test target is specified.
```

**Note:** `headers` attribute referencing to the cell itself are ignored when [assigning header cells (step 3, first case, point 2)](https://html.spec.whatwg.org/multipage/tables.html#algorithm-for-assigning-header-cells).

## Assumptions

- This rule assumes that the `headers` attribute is only used to identify table headers. If other information is included in the `headers` attribute, the rule may fail on issues that are not accessibility concerns. For example, if `headers` is used to include information for scripts, this rule may not be accurate.

> - This rule assumes that the `headers` attribute is required to express the relationship between data and table header cells in the same `table`. If the browser [computes an adequate fallback header][] for cells that have the `headers` attribute [attribute value][] that does not correspond to the value of the `id` attribute of any one cell in the same `table`, success Criterion [1.3.1 Info and Relationships][sc131] may be satisfied even if this rule failed.

Reviewer: `[sc131]` needs an entry in the references list.
